lectures/additive_functionals.md

```diff
@@ -1194,14 +1194,15 @@ Then let's use the plots to  investigate how these densities evolve through time
 
 We will plot the densities of $\log {\widetilde M}_t$ for different values of $t$.
 
-Note: `scipy.stats.lognorm` expects you to pass the standard deviation
+```{note}
+`scipy.stats.lognorm` expects you to pass the standard deviation
 first $(tH \cdot H)$ and then the exponent of the mean as a
 keyword argument `scale` (`scale=np.exp(-t * H2 / 2)`).
 
 * See the documentation [here](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.lognorm.html#scipy.stats.lognorm).
 
 This is peculiar, so make sure you are careful in working with the log normal distribution.
-
+```
 Here is some code that tackles these tasks
```

Comment:
@HumphreyYang super picky I know but I find it helpful to have a space after every directive -- make it a bit clearer to read the file.

Reply:
@mmcky Thanks for the kind style advice. It does look better with directives separated out from the text. I will keep this in mind. Could you have a quick look at the latest commits in both repos to see if it is what you expected?

Many thanks.